gateway/server.go

```diff
@@ -141,11 +141,12 @@ func main() {
 
 	r.HandleFunc("/system/alert", faasHandlers.Alert)
 
-	r.HandleFunc("/system/function/{name:[-a-zA-Z_0-9]+}", queryFunction).Methods(http.MethodGet)
+	r.HandleFunc("/system/function/{name:[-a-zA-Z_0-9]+}", queryFunction).
+		Methods(http.MethodGet, http.MethodHead, http.MethodOptions)
```

Comment:
Shouldn't this list be: `http.MethodPost http.MethodPut, http.MethodPatch, http.MethodDelete, http.MethodGet` ?